acceptance-tests/Makefile.am

```diff
@@ -1237,25 +1272,8 @@ CORECLR_DISABLED_TEST_CS_SRC += 	\
 CORECLR_DISABLED_TEST_CS_SRC += $(CORECLR_PATH)/tests/src/Exceptions/Finalization/Finalizer.cs
 
 # exclude the CoreCLR GC stress framework for now, it needs special integration:
-CORECLR_DISABLED_TEST_CS_SRC += 	\
-	$(CORECLR_PATH)/tests/src/GC/Stress/Tests/plug.cs	\
-	$(CORECLR_PATH)/tests/src/GC/Stress/Tests/SingLinkStay.cs	\
-	$(CORECLR_PATH)/tests/src/GC/Stress/Tests/pinstress.cs	\
-	$(CORECLR_PATH)/tests/src/GC/Stress/Tests/GCQueue.cs	\
-	$(CORECLR_PATH)/tests/src/GC/Stress/Tests/ExpandHeap.cs	\
-	$(CORECLR_PATH)/tests/src/GC/Stress/Tests/PlugGaps.cs	\
-	$(CORECLR_PATH)/tests/src/GC/Stress/Tests/concurrentspin2.cs	\
-	$(CORECLR_PATH)/tests/src/GC/Stress/Tests/ThdTreeGrowingObj.cs	\
-	$(CORECLR_PATH)/tests/src/GC/Stress/Tests/573277.cs	\
-	$(CORECLR_PATH)/tests/src/GC/Stress/Tests/GCVariant.cs	\
-	$(CORECLR_PATH)/tests/src/GC/Stress/Tests/bestfit-finalize.cs	\
-	$(CORECLR_PATH)/tests/src/GC/Stress/Tests/allocationwithpins.cs	\
-	$(CORECLR_PATH)/tests/src/GC/Stress/Tests/MulDimJagAry.cs	\
-	$(CORECLR_PATH)/tests/src/GC/Stress/Tests/GCSimulator.cs	\
-	$(CORECLR_PATH)/tests/src/GC/Stress/Tests/StressAllocator.cs	\
-	$(CORECLR_PATH)/tests/src/GC/Stress/Tests/doubLinkStay.cs	\
-	$(CORECLR_PATH)/tests/src/GC/Stress/Tests/LeakGenThrd.cs	\
-	$(CORECLR_PATH)/tests/src/GC/Stress/Tests/b115557.cs	\
+CORECLR_DISABLED_TEST_CS_SRC +=	\
+	$(CORECLR_STRESS_TESTS_SRC)	\
```

Comment:
`CORECLR_DISABLED_TEST_CS_SRC` is intended to list files that don't make sense to run but need to be included somewhere so we can easily check if new files were added upstream with `coreclr-list-missing-tests`.

Add it to `CORECLR_DEFINED_CS_SRC` below instead, since we're actually using the GC files.